src/Random/Pcg.elm

```diff
@@ -105,7 +105,7 @@ to set up anyway.
 generate : (a -> msg) -> Generator a -> Cmd msg
 generate toMsg generator =
     Time.now
-        |> Task.map (round >> initialSeed >> step generator >> Tuple.first)
+        |> Task.map (Time.posixToMillis >> toFloat >> round >> initialSeed >> step generator >> Tuple.first)
```

Comment:
Probably this `toFloat >>  round` doesn't make much sense, does it? I think I can remove them both? :thinking: